src/main/kotlin/com/wynntils/athena/routes/managers/ItemManager.kt

```diff
@@ -1,12 +1,16 @@
 package com.wynntils.athena.routes.managers
 
+import com.wynntils.athena.core.asJSON
 import com.wynntils.athena.core.utils.JSONOrderedObject
 import org.json.simple.JSONArray
 import org.json.simple.JSONObject
+import java.io.FileReader
 
 object ItemManager {
 
     fun convertItem(input: JSONObject): JSONOrderedObject {
+        val wynnbuilderIDs = FileReader(javaClass.getResource("wynnbuilderIDs.json").file).readText().asJSON<JSONObject>()
```

Comment:
It would be better to load this file once as opposed to every time convertItem is called

Reply:
Oh yeah good catch. I will change that.